subliminal/subtitle.py

```diff
@@ -52,38 +83,38 @@ def __init__(self, language, hearing_impaired=False, page_link=None, encoding=No
                 logger.debug('Unsupported encoding %s', encoding)
 
     @property
-    def id(self):
-        """Unique identifier of the subtitle"""
-        return ""
+    def id(self) -> str:
+        """Unique identifier of the subtitle."""
```

Comment:
aah, i just noticed - do we have a linter rule that enforces these dots? not sure if i like them

Reply:
it's [D400](https://docs.astral.sh/ruff/rules/ends-in-period/).
it's automatically fixed by ruff.